src/chainparams.cpp

```diff
@@ -353,7 +353,7 @@ class CTestNetParams : public CChainParams {
             uint256S("05688d8a0e9ff7c04f6f05e6d695dc5ab43b9c4803342d77ae360b2b27d2468e");
         consensus.vUpgrades[Consensus::UPGRADE_CANOPY].nProtocolVersion = 170012;
         consensus.vUpgrades[Consensus::UPGRADE_CANOPY].nActivationHeight =
-            Consensus::NetworkUpgrade::NO_ACTIVATION_HEIGHT;
+            Consensus::NetworkUpgrade::1020200;
```

Comment:
```suggestion
       consensus.vUpgrades[Consensus::UPGRADE_CANOPY].nActivationHeight = 1020500;
```

This is around 17:50Z on the 30th.